source/retryable-reads/tests/README.rst

```diff
@@ -150,6 +150,14 @@ the SDAM's ``minHeartbeatFrequencyMS`` from 500ms to 50ms, thus decreasing the
 waiting time after a "not master" error:
 
 .. _minHeartbeatFrequencyMS: https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#minheartbeatfrequencyms
+
+Unsupported commands
```

Comment:
Hm, would this be effectively a duplicate of the "Optional Enumeration Commands" section below?

Reply:
Perhaps the two sections could be combined under the heading "Unsupported optional enumeration commands" for greater clarity and minimal redundancy?

Reply:
You are correct. I missed that. I will close as the ticket as already fixed.